exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,11 @@
+var messageTypes = "My name is";
 const name = " Daniel  ";
-
+var nameTrim = name.trim();
+var messageType = "and my name is ";
+var nameLength = name.length;
+var many = " characters long.";
+var message = messageTypes + name + messageType + name.length + many;
```

Comment:
The whitespace is still present here as you've included the name var instead of the nameTrim